numpy/core/include/numpy/npy_3kcompat.h

```diff
@@ -219,35 +230,21 @@ npy_PyFile_Dup2(PyObject *file, char *mode, npy_off_t *orig_pos)
     if (handle == NULL) {
         PyErr_SetString(PyExc_IOError,
                         "Getting a FILE* from a Python file object failed");
+        return NULL;
```

Comment:
Depending on whether this PR makes 1.16, it might be worth putting this one line change in a separate PR.

Reply:
Yeah, i left it in it's own commit just for that.

I didn't find many test for `fromfile`. It is entirely possible that this particular bug always existed. but maybe I'm wrong.

Reply:
Thanks for doing that